File: src/com/wmp/countdown/tools/printLog/PrintLogStyle.java
package com.wmp.countdown.tools.printLog;

import java.awt.*;

public class PrintLogStyle {
    private LogStyle style;

    public PrintLogStyle(LogStyle style) {
        this.style = style;
    }

    public LogStyle getStyle() {
        return style;
    }

    public void setStyle(LogStyle style) {
        this.style = style;
    }

    public void print(String owner, String logInfo) {
        Log.print(style, owner, logInfo, null);
    }

    public void print(Container c, String owner, String logInfo) {
        Log.print(style, owner, logInfo, c);
    }
}

File: src/com/wmp/publicTools/printLog/PrintLogStyle.kt
package com.wmp.publicTools.printLog

import java.awt.Container

open class PrintLogStyle(@JvmField var style: LogStyle) {
    open fun print(owner: String, logInfo: Any) {
        Log.print(style, owner, logInfo, null)
    }

    open fun print(c: Container?, owner: String, logInfo: Any) {
        Log.print(style, owner, logInfo, c)
    }
}

